src/autopas/cells/ParticleCell.h

```diff
@@ -82,10 +84,22 @@ class ParticleCell {
   virtual void addParticle(const Particle &p) = 0;
 
   /**
-   * Get the number of particles stored in this cell.
-   * @return number of particles stored in this cell
+   * Get the number of all particles stored in this cell (owned, halo and dummy).
+   * @return number of particles stored in this cell (owned, halo and dummy).
    */
-  [[nodiscard]] virtual unsigned long numParticles() const = 0;
+  [[nodiscard]] virtual unsigned long size() const = 0;
+
+  /**
+   * Get the number of particles with respect to the specified IteratorBehavior.
+   * Note: Since this function counts the number of the respective particles in the internal particle storage, this is
```

Comment:
```suggestion
   * @warning Since this function counts the number of the respective particles in the internal particle storage, this is
```

Reply:
done in[37e2621](https://github.com/AutoPas/AutoPas/pull/782/commits/37e26218410380f7bc586d66bd83b316d23a3d28)